modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorSettingsTest.ts

```diff
@@ -98,7 +98,7 @@ describe('browser.tinymce.themes.silver.editor.color.ColorSettingsTest', () => {
     },
     {
       text: 'Pale tomato',
-      value: '#FFB0A2',
+      value: '#FF6347',
```

Comment:
What happened here?

Reply:
See the `Fixed` changelog entry, the old logic was calculating the RGBA value for hex incorrectly.